src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -133,7 +143,20 @@ internal static void SetLogLevel(LogEventLevel logLevel)
 
         internal static void UseDefaultLevel()
         {
-            SetLogLevel(LogEventLevel.Information);
+            SetLogLevel(DefaultLogLevel);
+        }
+
+        private static int GetRateLimit()
+        {
+            string rawRateLimit = EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.LogRateLimit);
+            if (!string.IsNullOrEmpty(rawRateLimit)
+                && int.TryParse(rawRateLimit, out var rate)
+                && (rate >= 1))
```

Comment:
```suggestion
                && (rate >= 0))
```
We should be able to return a rate of 0 to invoke the behavior of "Log every message"

Reply:
Also, is there a corresponding test that will read the environment variable and make sure that 0 logs everything whereas anything else does rate limiting? I see great unit testing on the individual rate limiting implementations, but didn't see one that uses the environment variable

Reply:
Reading through previous comments, I think this rate limiting was being conflated with log rate limiting. This limit is allowed to be `0` per the RFC and should allow all logs to be written

Reply:
🤦‍♂️ You're absolutely right, it should be `>= 0`. RE unit tests, no, we don't have any for this class at all due to the difficulty testing a static initialization. I'd definitely like to add some, but [as mentioned in this comment](https://github.com/DataDog/dd-trace-dotnet/pull/1153#discussion_r567790844), it would probably require refactoring this class to use a factory type instead, which seemed out of scope for this (already large) PR?